admin/server/projects.go

```diff
@@ -377,6 +403,19 @@ func (s *Server) AddProjectMember(ctx context.Context, req *adminv1.AddProjectMe
 		return nil, status.Error(codes.PermissionDenied, "not allowed to add project members")
 	}
 
+	count, err := s.admin.DB.CountOrganizationOutstandingInvitations(ctx, proj.OrganizationID)
+	if err != nil {
+		return nil, status.Error(codes.Internal, err.Error())
+	}
+	// TODO can be optimized by fetching org level quotas in FindProjectByName
+	org, err := s.admin.DB.FindOrganization(ctx, proj.OrganizationID)
+	if err != nil {
+		return nil, status.Error(codes.Internal, err.Error())
+	}
+	if count >= org.QuotaOutstandingInvitations {
+		return nil, status.Errorf(codes.FailedPrecondition, "cannot invite more users, org %q already %d outstanding invitations", org.Name, org.QuotaOutstandingInvitations)
+	}
```

Comment:
1. Can remove the TODO (many queries can theoretically be optimized, but let's wait until we see latency problems before optimizing). 
2. Add a comment at top of the block like other places (like `// Check outstanding invites quota`)